sentry-spring/src/main/java/io/sentry/spring/tracing/SentryTracingFilter.java

```diff
@@ -73,6 +73,11 @@ protected void doFilterInternal(
     if (hub.isEnabled()) {
       final String sentryTraceHeader = httpRequest.getHeader(SentryTraceHeader.SENTRY_TRACE_HEADER);
 
+      hub.configureScope(
```

Comment:
`ITransaction#setRequest` is deprecated now as these kind of properties should be set on the scope and then applied to transaction in `SentryClient`.